cmd/scw/testdata/test-all-usage-function-deploy-usage.golden

```diff
@@ -0,0 +1,22 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Create or fetch, upload and deploy your function
+
+USAGE:
+  scw function deploy [arg=value ...]
+
+ARGS:
+  [namespace-id]    Function Namespace ID to deploy to
+  name              Name of the function to deploy, will be used in namespace's name if no ID is provided
+  runtime            (unknown_runtime | golang | python | python3 | node8 | node10 | node14 | node16 | node17 | python37 | python38 | python39 | python310 | go113 | go117 | go118 | node18 | rust165 | go119 | python311 | php82 | node19 | go120)
```

Comment:
Remove `unknown_runtime`

Reply:
This is the runtime values of `function function create`, I don't plan on listing them manually, this list should be updated when expanded by generation